cinemas_repertoire/serializers.py

```diff
@@ -0,0 +1,88 @@
+from rest_framework import serializers
+from .models import (
+    AddressInfo,
+    CinemaCityCinema,
+    CinemaCityEvent,
+    CinemaCityMovie
+)
+
+
+class CinemaCityMovieAPIResponseSerializer(serializers.Serializer):
+    movie_id = serializers.CharField(source='id')
+    name = serializers.CharField()
+    length = serializers.IntegerField()
+    poster_link = serializers.URLField(source='posterLink')
+    video_link = serializers.URLField(source='videoLink')
+    link = serializers.URLField()
+    weight = serializers.IntegerField()
+    release_year = serializers.IntegerField()
+    attribute_ids = serializers.ListField(source='attributeIds')
+
+
+class CinemaCityEventAPIResponseSerializer(serializers.Serializer):
+    event_id = serializers.CharField(source='id')
+    film_id = serializers.CharField(source='filmId')
+    cinema_id = serializers.CharField(source='cinemaId')
+    business_day = serializers.CharField(source='businessDay')
+    event_datetime = serializers.DateTimeField(source='eventDateTime')
+    attribute_ids = serializers.ListField(source='attributeIds')
+    booking_link = serializers.URLField(source='bookingLink')
+    sold_out = serializers.BooleanField(source='soldOut')
+
+
+class AddressInfoAPIResponseSerializer(serializers.Serializer):
+    address_1 = serializers.CharField(source='address1', allow_null=True)
+    address_2 = serializers.CharField(source='address2', allow_null=True)
+    address_3 = serializers.CharField(source='address3', allow_null=True)
+    address_4 = serializers.CharField(source='address4', allow_null=True)
+    city = serializers.CharField()
+    state = serializers.CharField(allow_null=True)
+    postal_code = serializers.CharField(source='postalCode', allow_null=True)
+
+
+class CinemaCityCinemaAPIResponseSerializer(serializers.Serializer):
```

Comment:
How about making this model serializer? (it may require some custom fields, etc. but may be a good way to exercise ;) )